Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -88,6 +88,12 @@ public static void RunPreStartup(ServerApplicationPaths appPaths, ILoggerFactory
                  ? (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!
                  : new MigrationOptions();
 
+            // 10.10 specific EFCore migration check.
+            if (migrationOptions.Applied.Any(f => f.Id.ToString("D").Equals("36445464-849f-429f-9ad0-bb130efa0664", StringComparison.InvariantCulture) || f.Name == "MigrateLibraryDbData"))
```

Comment:
Should save the guid above and do a guid comparison instead
Also this comparison is failing build ci